### DISCUSSION TITLE
Duda sobre ordenamiento consulta IN-CIRCLE

### DISCUSSION THREAD

**Discussion by jtlledo** [IIC2133-PUC/2024-1#94] [Tarea 1] [answered]
Hola! Tengo una duda sobre el orden en que se deben entregar las cabezas en la consulta IN-CIRCLE. En el enunciado sale que se debe ser primero por x y luego por y. Esto se refiere a que las cabezas deben estar ordenadas de menor a mayor en x y luego en y, o se refiere a que debo revisar primero las 'capas x' de mi árbol 2d y luego las 'capas y'? Lo pregunto porque la forma en la que tengo implementada la consulta funciona, pero entrega en 'desorden'.

**Comment by sebacarrascop** [comment, marked as answer]
Deben estar ordenadas por x, en caso de que 2 o más cabezas tomen el mismo valor de x, entonces esas cabezas se deben ordenar por y. (Podrías verlo como que la llave para el orden es la tupla x,y). Con eso no hay problemas ya que no pueden haber más de una cabeza en el mismo x e y.
